bedrock/settings/base.py

```diff
@@ -406,7 +406,6 @@ def lazy_langs():
     "images",
     "contribute.json",
     "credits",
-    "gameon",
```

Comment:
**nit:** could we change the comment above here to something that no longer references `/gameon/`? Using `/media/` instead is probably a good example:

```
# Paths that don't require a locale code in the URL.
# matches the first url component (e.g. mozilla.org/media/)
```

Reply:
Not a nit, at all! How did I miss this? Great spot 👀 — thanks for really reviewing thoroughly, even the most trivial cleanups… Thanks! (Going with /credits as that's pretty much well known around…)